.gitignore

```diff
@@ -20,3 +20,11 @@ cabal.project.freeze
 *~
 *#
 .#*
+intTests/test_intro_examples/*.cnf
```

Comment:
How did you determine what files from `intTests` to put in `.gitignore`?

Reply:
Chatted w/Ryan, this is in reference to #1067.  @LisannaAtGalois , we could perhaps optimistically invoke a make clean after each target, ignoring failures (e.g. no clean target), but we should probably also make sure the gitignore us up-to-date with the latest test outputs.

Reply:
I'll take a look and manually update the .gitignore (if needed). Maybe this could be automatically linted in CI by running all the tests (or their makefiles?) and checking to see if the git tree is dirty.

Invoking make clean at least on test success makes sense to me. Probably better to have failing tests not invoke that to preserve the output for post-mortem analysis.